packages/react/src/lib/accessibility/Behaviors/Alert/alertWarningBehavior.ts

```diff
@@ -1,17 +1,21 @@
 import { Accessibility } from '../../types'
+import AlertBehaviorProps from './alertBehaviorProps'
 
 /**
  * @specification
- * Adds role 'alert' to 'content' slot.
- * Adds attribute 'aria-live=polite' to 'content' slot.
+ * Adds role 'alert' to 'wrapper' element.
+ * Adds attribute 'aria-live=polite' to 'wrpper' element.
  */
 
-const alertWarningBehavior: Accessibility = () => ({
+const alertWarningBehavior: Accessibility<AlertBehaviorProps> = props => ({
   attributes: {
-    content: {
+    wrapper: {
       role: 'alert',
       'aria-live': 'polite',
     },
+    dismissAction: {
+      'aria-labelledby': props.wrapperId,
```

Comment:
aria-describedby as label should be something like 'Dismiss' - that needs to come from outside as translations are out of scope for stardust

Reply:
@jurokapsiar 
I think it could be pointing to wrapperId.
But it need be there aria-describedby prop not "aria-labelledby"

Then the patter is, depend on close button:
if icon
<button aria-describedby="wrapperId" aria-label="dismiss"> </button>
if button has text
<button aria-describedby="wrapperId"> dismiss </button>